### PR TITLE
TcBSD PLC Swap / Minor grating pitch tune

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

- Set Grating deadband wider 20 nrad instead of 10.
- Rebuild with TwinCAT/BSD plc-rixs-optics
- 4 Cores allocated
- 3 dedicated Cores, one allocated to NC, PlcTask, and PlcDaq
- Needed to manually install TF6310-TCP-IP: 2.0.14.0_26 for daq UDP interface to work.
- Add sync groups for SL1K2 and SP1K1.

## Motivation and Context

- PLC CPU overloaded
- SP1K1 Grating Pitch Unstable

## How Has This Been Tested?

- RIX Optics all appear back online via the Lucid and PyDm screens.

## Where Has This Been Documented?

https://jira.slac.stanford.edu/browse/ECS-4002

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: lcls-plc-rixs-optics/_Config/NC/Axes/g_pi.xti [hunk 1]
```diff
@@ -1440,8 +1440,8 @@ External Setpoint Generation:
 		</Drive>
 		<Controller Name="Ctrl" CtrType="2">
 			<CtrPara PriorControlFactor="1">
-				<PosDiffControl Time="0.2"/>
```
Comment (nrwslac) [inline lcls-plc-rixs-optics/_Config/NC/Axes/g_pi.xti:1443]: @ZLLentz @ghalym I don't see anything in these lines that reflect what I thought I did, which was change the control loop from a PID  to Two Variable gain loop. Did I not do that, or is someone not version controlled?

Comment (nrwslac) [reply lcls-plc-rixs-optics/_Config/NC/Axes/g_pi.xti:1443]: Maybe that line 1441 CtrType="2", meaning I didn't actually change the control type.

Comment (ZLLentz) [reply lcls-plc-rixs-optics/_Config/NC/Axes/g_pi.xti:1443]: When I open the pre-PR version of the project, I see that we are already using the 2 variable control loop, not the PID loop (though some of the other axes are using the full PID loop)

**Review by ghalym — APPROVED** [review]
Approving so we can move forward with Exit slits work.

**Pull request merged**